include/spdlog/details/async_log_helper.h

```diff
@@ -218,7 +216,11 @@ inline spdlog::details::async_log_helper::~async_log_helper()
     try
     {
         push_msg(async_msg(async_msg_type::terminate));
```

Comment:
New approach: flush and wait for the thread pool to consume the queue.
Tell me what you think about this.